src/clj/game/cards-ice.clj

```diff
@@ -825,6 +825,23 @@
                  :cost [:credit 1] :effect (effect (add-prop target :advance-counter 1))}]
     :subroutines [(tag-trace 2)]}
 
+   "Mausolus"
+   {:advanceable :always
+    :subroutines [{:label "Gain 1 [Credits] (Gain 3 [Credits])"
+                   :msg (msg "gain " (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3) " [Credit]")
+                   :effect (effect (gain :credit (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3)))}
+                  {:label "Do 1 net damage (Do 3 net damage)"
+                   :delayed-completion true
+                   :msg (msg "do " (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3) " net damage")
+                   :effect (effect (damage eid :net (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3) {:card card}))}
+                  {:label "Give the Runner 1 tag (and end the run)"
+                   :delayed-completion true
+                   :msg (msg "give the Runner 1 tag"
+                          (when (>= (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) " and end the run"))
+                   :effect (req (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3)
+                                  (tag-runner state :runner 1)
+                                  (do (tag-runner state :runner 1) (end-run state side))))}]}
```

Comment:
This could be changed to trigger the `tag-runner` first, irrespective of advancement counters, and then trigger `end-run` if 3 or more counters, since the `req` has an implicit `do`.

```clojure
(req (tag-runner ...)
     (when (>= ... 3) 
       (end-run ...)))
```